src/controllers/userController.js

```diff
@@ -4,13 +4,16 @@ const {
   createUser,
   updateUserPassword,
 } = require('../services/userServices');
+const log4js = require('../logs');
+
+const logger = log4js.getLogger('usersController'); 
 
 const getAllUsersController = async (req, res, next) => {
   try {
     await getAllUsers(req, res);
     next();
   } catch (e) {
-    console.log(`Error: ${e}`);
+    logger.trace(e);
```

Comment:
Acá se loguearía un error con la traza.

[Ejemplo](https://stackoverflow.com/questions/4347797/how-to-send-a-stacktrace-to-log4j)